pyproject.toml

```diff
@@ -147,7 +147,7 @@ python-apt = { index = "python-apt-wheels" }
 
 [build-system]
 requires = [
-    "setuptools==75.9.1",
+    "setuptools==80.9.0",
```

Comment:
Do we care about https://github.com/canonical/rockcraft/blob/cd00edc2f2849b2107284edadede8fe4c1458ecc/pyproject.toml#L15 for testcraft and partitioncraft?

Reply:
I don't think so - moreover I'm not actually seeing those warnings for testcraft?

Reply:
Interesting, I'm not seeing it either! I was assuming it would crop up.